test/test.go

```diff
@@ -505,6 +505,10 @@ func (h *Helper) Path(name string) string {
 		joined = filepath.Join(h.tempdir, name)
 	}
 
+	if _, err := os.Stat(joined); os.IsNotExist(err) {
+		return ""
+	}
+
 	// Ensure it's the absolute, symlink-less path we're returning
 	abs, err := filepath.EvalSymlinks(joined)
```

Comment:
If i remove the check and run the manifest-exists test i get an error as before going into CompareFile it tries to get the path to the lock in
final.

Will run again and share the stack trace from the error in the morning.